slope/index.js

```diff
@@ -130,6 +132,7 @@ parseData.fromCSV('./data.csv', dateStructure).then(({ seriesNames, groupNames,
         myChart.dotRadius(currentFrame.rem() * 0.3);
 
         // specifics based on frame type
+        myChart.setPalette(setColourPalette);
         myChart.colourPalette(frameName, groupNames); // set colour palette
```

Comment:
This is a bit confusing — you set palette, then you set colour palette? It's not a very intuitive API if both these steps are needed separately.